### DISCUSSION TITLE
Send Notification just to group chat, not to user

### DISCUSSION THREAD

**Discussion by tbpoetke** [metalmatze/alertmanager-bot#153] [Help/Support] [answered]
looks like if i stop the bot at the direct chat it will also not alerting in group chat, is it possible to add the channel id, so bot will just post in group chat? Really annoying get messaged at 2 chats same time.

**Comment by avazxonov** [comment, marked as answer]
You can just click "stop and block bot" from telegram chat in bot menu. 
![Screenshot from 2020-07-22 15-25-42](https://user-images.githubusercontent.com/66010809/88165756-a3b07080-cc2f-11ea-9b16-91c0b20a71a9.png)
